content/terms_and_definitions/terms_and_definitions.adoc

```diff
@@ -6,7 +6,86 @@ ifndef::include-only-once[]
+
+Material assignment::
+A mapping of material IDs to parts of 3D models.
+The material IDs are unique a point to a set of material properties in separate material files.
+
+Material classification::
+Standardized classification scheme that sorts materials into classes and groups of classes, for example, based on their physical properties.
+
+Material Parameter File::
+A file in json format containing defined material properties according to the OpenMATERIAL standard.
+
+Material property::
+Intensive physical or chemical properties of a material 
+
+* Notes: Open Point: What about geometric properties, i.e. roughness, thickness?
+
+Multispectral/hyperspectral::
+A wide range of wavelengths.
+
+Normals (face, vertex normals)::
+A vector that is perpendicular to a given object, surface, vertex or geometrical primitive.
+
+Object::
+An entity to which an object ID can be assigned.
+It comprises discrete meshes and defined 3D boundaries.
+
+Object Part / Subtype::
+(To be defined)
+
+* Notes: What is an object and what is an object part?
+
+Object Type::
+Class of objects with the same set of properties. Each instance of the class is described by the same properties, but has individual property values.
+
+Perception::
+The process of detecting objects and generating situational awareness of the surrounding of an automated system using perception sensors.
+"Machine perception is to reliably detect all the other road users relevant to the operation of the automated driving, and to assign them correctly to the traffic infrastructure."
+
+* Source: Klaus Dietmayer, Predicting of Machine Perception for Automated Driving, 2016
+
+Physical (as in physical parameter)::
+A physical property is any property of a physical system that is measurable. 
+
+* Source: Mark, Burgin (2016-10-27). Theory Of Knowledge: Structures And Processes
+
+Post (Rendering) Effects::
+Rendering techniques to improve fidelity while maintaining performance, that are typically non-physical (or in other words fake)
+
+Scenario::
+A description of the behavior or temporal evolution of physical objects and environmental conditions on the driving infrastructure over an interval of time, including the movement of traffic participants or the change of environmental conditions.
+
+* Source: ASAM OpenSCENARIO v1.3.0 standard, 2024
+
+Signal (Energy) Propagation::
+"During signal [or energy] propagation, the signal travels through the environment and possibly interacts with objects and particles."
+
+* Source: Linnhoff et al., Towards Serious Perception Sensor Simulation for Safety Validation of Automated Driving - A Collaborative Method to Specify Sensor Models, 2021
+
+Simulation Platform::
+Application orchestrating and running the simulation. The can be an all-in-one solution including the generation of the 3D environment or a co-simulation platform orchestrating a co-simulation of multiple sensor, models, vehicle dynamics models etc.
```

Comment:
Proposal:
Application that orchestrates and runs the simulations. It can be either an all-in-one solution including the generation of the 3D environment or a co-simulation platform orchestrating a co-simulation of multiple sensors, models, vehicle dynamics models, and so on.

Reply:
done